### ISSUE TITLE
Keep Screen On

### ISSUE THREAD

**Issue by NripPrajapat** [keymapperorg/KeyMapper#1367]
Add a trigger to keep the screen on.

**Comment by sds100** [comment]
Can you explain more? You want an action that sets the screen timeout to infinity?

**Comment by NripPrajapat** [comment]
> Can you explain more? You want an action that sets the screen timeout to infinity?

Yes, I guess, because most android OEMs stop activities after some time limit to save on battery, so it'll help the device not go in sleep mode.
So add a trigger that allows a user to set KEEP SCREEN ON(For Infinity, I guess) and another to TURN IT BACK TO NORMAL/DEFAULT.

**Comment by sds100** [comment]
What is your aim with this feature? It sounds like you're having another problem that you are trying to solve by doing this.

**Comment by NripPrajapat** [comment]
To keep the device on, to beat OEMs battery management and help user achieve the goals, like if someone is using an app, let's take Calculator for an example or notes, it will be like 'gaze detection' feature, which will give user, extra amount of time to be comfortable when using the app or doing other things without worrying about screen timeout.

**Comment by jambl3r** [comment]
seems out of scope and not particularly safe
